docs/medical-api/api-reference/fhir/get-medical-record-summary-status.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Get Medical Record Summary Status"
+description: "Check if a Medical Record Summary for a patient already exists, and when it was generated."
+api: "GET /patient/:id/medical-record-status"
+---
+
+This endpoint checks whether the patient's Medical Record Summary already exists in the PDF and HTML formats.
+
+To create a Medical Record Summary, you need to use the [Start Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post).
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Response
+
+Returns a JSON containing the booleans for the existence of the Medical Record Summary in PDF and HTML formats, as well as the dates when they were generated.
+
+<ResponseExample>
+
+```javascript Metriport SDK
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+
+const metriport = new MetriportMedicalApi("YOUR_API_KEY");
+
+const status = await metriport.getMedicalRecordSummaryStatus(
+  "018a80c4-292a-7486-a223-6dcbc636c44c"
+);
+```
+
+</ResponseExample>
+
+<ResponseField name="pdf" type="boolean" required>
+  <Snippet file="medical-record-status.mdx" />
+</ResponseField>
+
+<ResponseField name="html" type="boolean" required>
+  <Snippet file="medical-record-status.mdx" />
+</ResponseField>
+
+```json
+{
+  "pdf": {
+    "exists": true,
+    "date": "2024-01-18T00:27:16.000Z"
+  },
+  "html": {
+    "exists": true,
+    "date": "2024-01-18T00:26:54.000Z"
+  }
+}
```

Comment:
IMO this interface is redundant, we will never have an MR summary that doesn't exist but was generated, right? Perhaps we can just remove the `exists` field entirely?

Reply:
I discussed this with @leite08 . The reason we keep this is because .headObject() from S3 *may* return an existing object with the `LastModified` field being undefined. I'm just trying to avoid cases where `date: undefined` for an existing file would result in the caller thinking the file does not exist

Reply:
i.e. according to their docs, it is possible for us to have 
`"html": { "exists": true, "date": undefined }`

Reply:
Alternativelly we can remove `exists` and set `updatedAt` to be a date way in the past? The first MAPI was in operation? 😄

Reply:
Concretely, what I'm getting at is our interface shouldn't be defined based on the underlying implementation @RamilGaripov - our customers don't care about S3, and we shouldn't make things more complicated b/c of it.

Since the only thing customers care about is the date the file was created at, I suggest we make the interface reflect that.